src/lib/djerba/core/html/research_footer.html

```diff
@@ -0,0 +1,6 @@
+<p>
+  All information in this report is to be used for research only. No information in this report has been accredited by any accrediting bodies. Clinical interpretation of results was not performed. 
```

Comment:
Maybe change: "No information in this report has been accredited by any accrediting bodies."
To: "The process used to generate this report has not been certified by any accrediting bodies."

(Technically I think accreditation applies to the assay, not information in the individual report)